Sming/Core/Network/Http/HttpRequest.h

```diff
@@ -248,13 +248,11 @@ class HttpRequest
 		return this;
 	}
 
-#ifndef SMING_RELEASE
 	/**
 	 * @brief Tries to present a readable version of the current request values
 	 * @retval String
 	 */
 	String toString();
```

Comment:
Should also add a `String toString(const HttpRequest& req)` function for consistency with other APIs.